jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/ManagementUtils.java

```diff
@@ -892,7 +897,12 @@ private Component(String objectNamePattern, Collection<? extends T> beans) {
 			checkNames(beans, pattern);
 			/*[ENDIF]*/
 
-			this.beansByName = new HashMap<>(beans.size());
+			/*[IF JAVA_SPEC_VERSION >= 19]
+			this.beansByName = HashMap.newHashMap(beans.size());
+			/*[ELSE] JAVA_SPEC_VERSION >= 19 */
```

Comment:
This is not used: see line 604: `/*[IF !Sidecar19-SE]*/`.

Reply:
Removed Java 19 path.